### ISSUE TITLE
Outdated news on landing page

### ISSUE THREAD

**Issue by bianchini88** [ELIXIR-NO/website#37]
The landing page currently displays outdated news (see screenshot).

![image](https://github.com/user-attachments/assets/c023b077-a0d3-4532-8f25-e0dea553bcfd)

The actual [news page](https://website-70w.pages.dev/news/) is currently up-to-date. 
Would it be possible to fetch the most recent entries of the page instead of outdate items?
@yasinmiran

Tagging also @ingeborgwinge and @jonlaerdahl, as I believe showing our most recent activities on the landing page would be particularly useful.

**Comment by ingeborgwinge** [comment]
I agree, we should have the most recent at the top. Have we added any new updates recently?

**Comment by bianchini88** [comment]
@ingeborgwinge No updates since October, unfortunately (and there is plenty of stuff we could add, in principle...).

**Comment by ingeborgwinge** [comment]
We have agreed with Inge to have the person in charge of newsletter/webpages of Department of Informatics, UiB, to help us with those things in a 10% position. However, I think we should have the page up and running before we let her in. But then ideally we can send posts to her to put on the webpage.
